dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/telnet/InvokeTelnetHandler.java

```diff
@@ -150,6 +163,17 @@ public String telnet(Channel channel, String message) {
         return buf.toString();
     }
 
+    private Invoker<?> findInvoker(ProviderModel provider) {
+        for (Exporter<?> exporter : DubboProtocol.getDubboProtocol().getExporters()) {
```

Comment:
this change makes an assumption: all services will be exported with dubbo protocol. 

I am wondering if we should introduce this limitation, in order to make the filter work.

Reply:
should we try to find invoker from dubbo protocol first, and if there's no invoke found, then fallback to call the found method instead?